app/models/role_payment_type.rb

```diff
@@ -3,6 +3,8 @@ class RolePaymentType
 
   attr_accessor :id, :name
 
-  Unpaied = create!(id: 1, name: "Unpaid")
+  Consultant = create!(id: 4, name: "Paid as a consultant")
   ParliamentarySecretary = create!(id: 2, name: "Paid as a Parliamentary Secretary")
+  Whip = create!(id: 3, name: "Paid as a whip")
+  Unpaied = create!(id: 1, name: "Unpaid")
```

Comment:
Are we able to fix this typo? (I know you didn't introduce it)

Reply:
Good catch 👀 Thank you. Done.